lib/msal-common/src/cache/ICacheStorage.ts

```diff
@@ -8,12 +8,14 @@
  */
 export interface ICacheStorage {
     /**
-     * Function to read serialized cache from disk
+     * Function to read serialized Cache from disk
+     * @param key
+     * @param value
```

Comment:
nit: getSerializedCache() doesn't have parameters.

Reply:
Intentional. It is a getter to avoid access to private variables.

Reply:
We should remove the comments here.